### PR TITLE
Show block hash of a transaction

### PR DESCRIPTION
**Pull request by doasync**

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -3,3 +3,4 @@ TSC_COMPILE_ON_ERROR=false
 PORT=3005
 REACT_APP_ROWS_ON_PAGE=10
 REACT_APP_NETWORKS=[{"type": "TESTNET", "url": "wss://testnet-node-1.cere.network:9945", "label": "Testnet"},{"type": "TESTNET_DEV", "url": "wss://testnet-node-1.dev.cere.network:9945", "label": "Testnet DEV"}]
+REACT_APP_BLOCK_VIEWER_URL=https://block-viewer.cere.network
```
Comment (AndreiNavoichyk) [inline .env.dev:6]: What about other envs?

File: src/components/accountTransactions/AccountTransactions.tsx [hunk 2]
```diff
@@ -93,6 +95,10 @@ const AccountTransactions = (
                   <tr key={index}>
                     <td scope="row">{index + txOnPage * (currentPage - 1) + 1}</td>
                     <td>{item.transactionHash}</td>
+                    <td>{blockViewerUrl
+                      ? <a href={`${blockViewerUrl}/#/explorer/query/${item.blockHash}`} target="_blank">{item.blockHash}</a>
```
Comment (AndreiNavoichyk) [inline src/components/accountTransactions/AccountTransactions.tsx:99]: We should support different Networks (DEV and PROD now). Do you handle this case?

File: .env.dev (hunk 1 above)
Comment (doasync) [reply .env.dev:6]: Yeah, I have [added](https://github.com/Cerebellum-Network/laboratory-ui/pull/9/commits/6224367c0bf02133c5e05671778e89e72654b29f) the variable to other environments

File: src/components/accountTransactions/AccountTransactions.tsx (hunk 2 above)
Comment (doasync) [reply src/components/accountTransactions/AccountTransactions.tsx]: I have added an `rpc` param

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**